index.bs

```diff
@@ -965,6 +977,44 @@ default-reader-asynciterator-prototype-internal-slots">Asynchronous iteration</h
  1. Return [=a promise resolved with=] undefined.
 </div>
 
+<h4 id="rs-transfer">Transfer via postMessage()</h4>
+
+<dl class="domintro">
+ <dt><code>worker.postMessage(rs, [rs]);</code>
```

Comment:
```suggestion
 <dt><code>destination.postMessage(rs, [rs]);</code>
```

Reply:
Alternately you could make it a bit more obvious with the new overload: `destination.postMessage(rs, { transfer: [rs] })`

Reply:
Good idea. Done.